package-lock.json

```diff
@@ -1,7 +1,10346 @@
 {
   "name": "sscs-cor-frontend",
   "version": "0.0.1",
+<<<<<<< HEAD
```

Comment:
this seems like it was corrupted in a merge conflict
also, I don't think we need both package-lock and yarn.lock files. we're supposed to be using yarn, so we shouldn't really have package-lock